src/features/flows/thingy91x/index.ts

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import Verify from '../../../common/steps/91FamilyVerify';
+import Apps from '../../../common/steps/Apps';
+import Develop from '../../../common/steps/develop';
+import Evaluate from '../../../common/steps/Evaluate';
+import Info from '../../../common/steps/Info';
+import Learn from '../../../common/steps/Learn';
+import Program from '../../../common/steps/program';
+import Rename from '../../../common/steps/Rename';
+import { Choice } from '../../device/deviceSlice';
+import SIM from './SIM';
+
+const infoConfig = {
+    title: 'Massive IoT Powerhouse',
+    markdownContent:
+        '![Thingy:91 X](Thingy91X.png)  \n&nbsp;  \nThe Nordic Thingy:91 X Prototyping Platform is perfect for running Cellular IoT or DECT NR+ prototypes.  \n&nbsp;  \nIt includes a nRF7002 (Wi-Fi IC), nRF5340 (short range IC) and all the necessary external circuirty like (e)SIM interface, antennas, programmable button and LEDs, and module interfaces.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/category/nrf-91-series)  \n&nbsp;  \n![9151 Cores](9151Cores.png)  \nYou have two options for leveraging the nRF9151 SiP:  \n&nbsp;  \n**Option 1** *Recommended*  \nHarness the full potential of nRF9151 by using the entire application core for your code, eliminating the need for an external MCU, and fully capitalizing on the advantages offered by nRF9151.  \n&nbsp;  \n**Option 2**  \nIntegrate nRF9151 as a [cellular](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/What-is-cellular-IoT#infotabs) or [NR+](https://www.nordicsemi.com/Products/DECT-NR) modem alongside your existing design running a serial AT command set on the application core.',
+};
+
+const programConfig = [
+    {
+        name: 'Prototyping explorer',
+        type: 'buttonless-dfu',
+        description:
+            'Connect to hello.nrfcloud.com to retrive real-time data and do simple prototyping.',
+        documentation: {
+            label: 'hello-nrfcloud',
+            href: 'https://github.com/hello-nrfcloud/firmware',
+        },
+        firmware: [
+            {
+                core: 'Application',
+                file: 'thingy91x_hello.nrfcloud.zip',
+                link: {
+                    label: 'hello-nrfcloud',
+                    href: 'https://github.com/hello-nrfcloud/firmware',
```

Comment:
```suggestion
                    href: 'https://github.com/hello-nrfcloud/firmware/blob/main/README.md',
```
There will be a doc page, but not sure if in time for release. We might have to update this later on.

Reply:
So I update it now?

Reply:
Yes, the link I'm suggesting is not to a doc page :)